js/src/forum/components/DiscussionPage.js

```diff
@@ -120,6 +120,10 @@ export default class DiscussionPage extends Page {
 
     if (this.discussion) {
       app.setTitle(this.discussion.title());
+      const firstPost = this.discussion.firstPost();
+      if (firstPost) {
+        app.setDescription(firstPost.contentPlain());
```

Comment:
Isn't this going to be very long for most posts?

Reply:
I've searched our codebase `js` but there's no `setDescription` method anywhere, where exactly is this supposed to have an effect?

Reply:
@franzliedke isn't setDescription truncating anyway?

Reply:
Oh, yes. :see_no_evil: